sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+    - sdk/resourcemanager/workloadssapvirtualinstance/armworkloadssapvirtualinstance/
+
+stages:
```

Comment:
```suggestion
extends:
```

Reply:
@Alancere this should fix your issue. We had to switch to extends recently to be compatible with newly required pipeline templates.